test/unit/logic/delegate.js

```diff
@@ -473,46 +461,39 @@ describe('delegate', function () {
 	describe('checkConfirmed', function () {
 
 		var validUsername;
-		var checkDuplicatesStub;
-		var transactionsExceptionsIndexOfStub;
+		var transactionsExceptionsIndexOfSpy;
 
 		beforeEach(function () {
 			validUsername = validSender.username;
-			checkDuplicatesStub = sinonSandbox.stub(delegate, 'checkDuplicates').callsArg(3);
-			transactionsExceptionsIndexOfStub = sinonSandbox.spy(exceptions.delegates, 'indexOf');
-		});
-
-		afterEach(function () {
-			transactionsExceptionsIndexOfStub.restore();
-			checkDuplicatesStub.restore();
+			sinonSandbox.stub(delegate, 'checkDuplicates').callsArg(3);
+			transactionsExceptionsIndexOfSpy = sinonSandbox.spy(exceptions.delegates, 'indexOf');
```

Comment:
Actually the pattern we're using for stubs (where we don't store the stub in a new variable if it's overriding an existing method) applies also to spies.